circle.yml

```diff
@@ -77,6 +77,12 @@ deployment:
           docker push ${DOCKER_ORGANIZATION:-$DOCKER_USER}/scope:$(./tools/image-tag) &&
           (test -z "${UI_BUCKET_KEY_ID}" || make ui-upload)
         )
+      - | 
+        test -z "${QUAY_USER}" || (
+          docker docker login -e '.' -u "$QUAY_USER" -p "$QUAY_PASSWORD" quay.io &&
+          docker tag weaveworks/scope:$(./tools/image-tag) "quay.io/${QUAY_ORGANIZATION}/scope:$(./tools/image-tag)" &&
+          docker push "quay.io/${QUAY_ORGANIZATION}/scope:$(./tools/image-tag)"
```

Comment:
It was on purpose, we are doing this to mitigate the 500s in WeaveCloud. However, if you have a use for it, I can also incorporate the `:latest` tag.